BE/deploy/scripts/blue-green-deploy.sh

```
@@ -0,0 +1,103 @@
+#!/bin/bash
+set -e
+
+# Blue/Green Deployment Script for ShortLink
+# This script performs zero-downtime deployment on a single EC2 instance
+
+ECR_REGISTRY="${ECR_REGISTRY}"
+IMAGE_TAG="${IMAGE_TAG:-latest}"
+NGINX_CONF="deploy/nginx/nginx.conf"
```

Shellcheck (0.11.0)

[warning] NGINX_CONF appears unused. Verify use (or export if used externally).

(SC2034)